docs/modules/migrate/pages/dmt-view-data.adoc

```diff
@@ -0,0 +1,32 @@
+= Verify the Migrated Data
+:description: To ensure that the migration includes all data, you can verify the size of the map in the target cluster.
```

Comment:
with the addition of CP migration support, we may want to update this page with more generic "data structure" migration validation. it can be either

- map
- replicated map 
- cp structures

for map and replicated map, size check should be enough. for cp structures, checking the existence of such a cp structure with the provided name should be good enough

Reply:
As I know, clc does not support replicated map, atomic reference, count down latch.

We can direct users to management center for those.

Reply:
[e3b6c3b](https://github.com/hazelcast/hz-docs/pull/936/commits/e3b6c3b17d71b48118a3cf9c9e2e607c215ed112)